docs/examples/layout/index.scss

```diff
@@ -19,15 +19,15 @@
     background-color: #18191a;
   }
 
-  .bg-purple-light {
+  .ep-bg-purple-light {
```

Comment:
A suggestion, I think we can change it to the form of css var by the way.

```scss
:root {
  --ep-c-bg-purple: xxx;
}

.dark {
  --ep-c-bg-purple: xxx;
}

.ep-bg-purple {
  background: var(--ep-c-bg-purple);
}
```

Reply:
completed.🥰